### DISCUSSION TITLE
RaspberryMatic-3.71.6.20230704-43388f-rpi3 - keine Übersetzung

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2354] [🇩🇪 Fragen&Antworten] [answered]
![Nowy obraz mapy bitowej](https://github.com/jens-maus/RaspberryMatic/assets/81189856/593c0234-69c4-4f4a-b4a7-0c725d0aa432)

**Comment by Baxxy13** [comment]
Mit deutscher UI sieht das soweit korrekt aus.
![RM_Feld-Tester](https://github.com/jens-maus/RaspberryMatic/assets/54898249/dc8bd145-22bf-4a97-b1dc-98d6463b82b9)

**Reply by ghost** [reply, marked as answer]
Danke Jeans-Maus :-)
![1](https://github.com/jens-maus/RaspberryMatic/assets/81189856/904e1aff-58bc-439e-a505-67f48f48d660)

**Comment by ghost** [comment]
![22](https://github.com/jens-maus/RaspberryMatic/assets/81189856/af34c866-ee32-4817-9997-b184d694ce55)

gilt für : RaspberryMatic-3.71.8.20230708-662cee-rpi3
